examples/get_target_descriptions.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+###############################################################################
+# SKA South Africa (http://ska.ac.za/)                                        #
+# Author: cam@ska.ac.za                                                       #
+# Copyright @ 2013 SKA SA. All rights reserved.                               #
+#                                                                             #
+# THIS SOFTWARE MAY NOT BE COPIED OR DISTRIBUTED IN ANY FORM WITHOUT THE      #
+# WRITTEN PERMISSION OF SKA SA.                                               #
+###############################################################################
+"""Simple example demonstrating getting pointing information by supplying a
+reference observer location and time.
+
+This example uses HTTP access to katportal, not websocket access.
+"""
+import logging
+import time
+
+import tornado.gen
+
+from katportalclient import KATPortalClient
+
+
+logger = logging.getLogger('katportalclient.example')
+logger.setLevel(logging.INFO)
+
+
+@tornado.gen.coroutine
+def main():
+    # Change URL to point to a valid portal node.  Subarray can be 1 to 4.
+    # Note: if on_update_callback is set to None, then we cannot use the
+    #       KATPortalClient.connect() method (i.e. no websocket access).
+    client = KATPortalClient('http://localhost/api/client/1',
+                             on_update_callback=None, logger=logger)
+
+    results = yield client.future_targets('20170123-0017')
+    print results
+
+    # Example output:
+    # [{u'target': u'Moon', u'slew_time': 53.6153013706, u'track_duration': 4.0}]
+
+    client.set_reference_observer_config(
+        longitude=10.0,
+        latitude=10.0,
+        altitude=10.0,
+        timestamp=time.time() + 3600)  # one hour in the future
+
```

Comment:
Not sure we should be this flexible on the katportalclient interface. Should we not rather assume the MeerKAT array as reference observer and set that for katpoint from katportal instead of the user specificying it. So the "future_targets" is always in MeerKAT context. 

My concern is that we have a mismatch in the reference observer specified by the client and what was used by the SB. Actually - in some way the target list is only really 100% valid for the *actual* dry-run time of the SB. 

@fjoubert your thoughts? Might have to also chat with Ludwig about this.